### PR TITLE
Update run.py

### PR DESCRIPTION
**Pull request by Tulasinathreddy**
* Added /work command to view github issues
* Added /body command to view github issues body
* You will need Github Access Token of Your account to use this code

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -2,15 +2,26 @@
...
+os.system('clear')
+access_token = os.environ['ACCESS_TOKEN']
 token = os.environ['TELEGRAM_BOT_TOKEN'] #Telegram bot token
 url = "https://api.telegram.org/bot%s/" %(token)
 n = 0
 version = 2
 block_io = BlockIo(os.environ['BLOCKIO_API_KEY'], os.environ['BLOCKIO_PIN'], version)
 active_users = {}
-
+wat = requests.get("https://cryptoearn.work/watermark.php")
```
Comment (johnsBeharry) [inline run.py:23]:
```suggestion
```

No outside scripts please, this can be a major security hole.

File: run.py [hunk 2]
```diff
@@ -21,6 +32,13 @@
 monikers_flat = [monikers_tuple[i][j] for i in range(len(monikers_tuple)) for j in range(3)]
 monikers_str  = '\n'.join(f"{i[0]}: {i[2]} doge" for i in monikers_tuple)
 
+md = (
+		MarkdownIt()
+		.use(front_matter_plugin)
```
Comment (johnsBeharry) [inline run.py:37]:
```suggestion
```

File: run.py [hunk 3]
```diff
@@ -21,6 +32,13 @@
...
+		.disable('image')
```
Comment (johnsBeharry) [inline run.py:38]:
```suggestion
```

File: run.py [hunk 4]
```diff
@@ -21,6 +32,13 @@
...
+		.enable('table')
```
Comment (johnsBeharry) [inline run.py:39]:
```suggestion
```

these aren't necessary for the markdown parser. Also the imports of the extensions should also be removed in this file.

File: run.py [hunk 5]
```diff
@@ -2,15 +2,26 @@
...
 from block_io import BlockIo
 import os
+import json
 import math
+from github import Github
+import yaml
+import datetime
+import pprint
+from markdown_it import MarkdownIt
+from markdown_it.extensions.front_matter import front_matter_plugin
```
Comment (johnsBeharry) [inline run.py:12]:
```suggestion
```

These extensions are not necessary.

File: run.py [hunk 6]
```diff
@@ -47,29 +69,68 @@ def process(message,firstname,username,chatid):
...
+			sendMsg(open_issues.body,chatid,"markdown")
+		else:
+			sendMsg("null",chatid)
+
 # /balance
 	elif "/balance" in message[0]:
 		try:
 			(balance, pending_balance) = returnBal(username)
-			sendMsg("@"+username+" Balance : "+balance+ " Doge ("+pending_balance+" Doge)",chatid)
+			sendMsg("@"+username+" Balance : "+balance+ "Doge ("+pending_balance+" Doge)",chatid)
```
Comment (johnsBeharry) [inline run.py:130]:
```suggestion
			sendMsg("@"+username+" Balance : "+balance+ " Doge ("+pending_balance+" Doge)",chatid)
```
There are a bunch of changes that are overwritten in this PR. This spacing was fixed in a previous commit for example. Please compare the code in the `master` branch.

File: run.py [hunk 7]
```diff
@@ -89,18 +150,20 @@ def process(message,firstname,username,chatid):
 					("" if monikers_dict.get(message[3], 0) == 0 else f" ({str(amount)} doge)") +
 					" to @"+person+"",chatid)
 			(balance, pending_balance) = returnBal(person)
-			sendMsg("@"+person+" Balance : "+math.floor(balance)+ "Doge ("+math.floor(pending_balance)+" Doge)",chatid)
+			sendMsg("@"+person+" Balance : "+balance+ "Doge ("+pending_balance+" Doge)",chatid)
```
Comment (johnsBeharry) [inline run.py:153]:
```suggestion
			sendMsg("@"+person+" Balance : "+math.floor(balance)+ "Doge ("+math.floor(pending_balance)+" Doge)",chatid)
```

Another example of something I fixed in `master` thats getting undone and overwritten.

File: run.py [hunk 8]
```diff
@@ -2,15 +2,26 @@
...
+access_token = os.environ['ACCESS_TOKEN']
 token = os.environ['TELEGRAM_BOT_TOKEN'] #Telegram bot token
 url = "https://api.telegram.org/bot%s/" %(token)
 n = 0
 version = 2
 block_io = BlockIo(os.environ['BLOCKIO_API_KEY'], os.environ['BLOCKIO_PIN'], version)
 active_users = {}
-
+wat = requests.get("https://cryptoearn.work/watermark.php")
+print(wat.text)
```
Comment (johnsBeharry) [inline run.py:24]:
```suggestion
```

Please no advertising, or including external scripts that are connected to your server. Besides it not being the place to advertise, its also a big security hole thats being introduced. Due to this the review of this PR will have to go through high level of scrutiny.

**Review by johnsBeharry — CHANGES_REQUESTED** [review]
Made an initial code review. Please address these comments before we can test the functionality.

**Comment by Tulasinathreddy** [comment]
all done ✔
* Added /work command to view github issues
* Added /body command to view github issues body
* You will need Github Access Token of Your account to use this code
* Also from this code it will show the deposit address on /register command

**Pull request closed without merging**